### PR TITLE
Updated gjtk package version (to make it work with latest io.js)

### PR DESCRIPTION
**Pull request by teologov**
Hi, this pull request is a part of https://github.com/balderdashy/waterline/issues/1080 issue

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,9 +31,9 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "dependencies": {
-    "validator": "~3.41.2",
+    "gjtk": "^1.0.3-beta",
```
Comment (dmarcelino) [inline package.json]: Does it only work with a beta version?

**Comment by dmarcelino** [comment]
Left an inline question, I'm not too keen on releasing waterline builds using beta dependencies... Can we avoid that?

**Comment by teologov** [comment]
Well, if I am not mistaken, the previous was a beta version as well. 
This is the latest version available in NPM.

**Comment by dmarcelino** [comment]
I've just did a `npm info gjtk` and the result was:

```
{ name: 'gjtk',
  description: 'GeoJSON ToolKit',
  'dist-tags': { beta: '1.0.0-b', latest: '1.0.3-beta' },
  versions: 
   [ '1.0.0-b',
     '1.0.1-beta',
     '1.0.2-beta',
     '1.0.3-beta' ],
//...
```

So yeah, there are no non-beta versions :)

**Pull request closed without merging**